roles/addons/nic_nmcli/tasks/main.yml

```diff
@@ -19,79 +19,54 @@
         - "vars/{{ ansible_facts.os_family }}.yml"
       skip: true
 
-- name: Package
+- name: "package █ Install {{ nic_nmcli_packages_to_install | join(' ') }}"
   package:
     name: "{{ nic_nmcli_packages_to_install }}"
     state: present
 
-- name: Enable/disable services
+- name: "service █ Manage {{ nic_nmcli_services_to_start | join(' ') }} state"
   service:
     name: "{{ item }}"
     enabled: "{{ (enable_services | bool) | ternary('yes','no') }}"
-  loop: "{{ nic_nmcli_services_to_start }}"
-  tags:
-    - service
-
-- name: Start services
-  service:
-    name: "{{ item }}"
     state: started
   loop: "{{ nic_nmcli_services_to_start }}"
-  when: (start_services | bool)
   tags:
     - service
 
-- name: Solve possible unmanaged (Ubuntu only)
-  file:
-    path: /etc/NetworkManager/conf.d/10-globally-managed-devices.conf
-    state: touch
-  when:
-    - ansible_facts.distribution_major_version == "18"
-    - ansible_facts.os_family == "Ubuntu"
+#- name: Solve possible unmanaged (Ubuntu only)
+#  file:
+#    path: /etc/NetworkManager/conf.d/10-globally-managed-devices.conf
+#    state: touch
+#  when:
+#    - ansible_facts.distribution_major_version == "18"
+#    - ansible_facts.os_family == "Ubuntu"
 
 # Note from JK
 # We could add so much more features here, like VLAN, LACP, MTU, etc. Easy to add.
-- name: Set NIC configuration
+- name: nmcli █ Set NIC configuration using nmcli
```

Comment:
Isn't it redundant?

Reply:
yes, you are right. I will correct this.